backend/Dockerfile

```diff
@@ -6,26 +6,31 @@ EXPOSE 443
 
 FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
 
+RUN mkdir -p /var/www/.nuget/packages; chown -R www-data:www-data /var/www
+USER www-data:www-data
```

Comment:
Why this? The docs show just putting user at the end and that should be all we need.

Reply:
Because I was getting mixed up between the main dockerfile and the dev dockerfile. I was thinking about the `dotnet watch` process not being able to overwrite files that were built as root — but that's not a concern here. I'll revert this and go back to the simple version.

Reply:
Reverted by force-pushing branch back to an earlier commit, then added commit https://github.com/sillsdev/languageforge-lexbox/commit/709df147e55e983a1f8dae3cebb4f4b00ec831e3 to use usernames rather than numeric UIDs as Chris suggested.